src/goto-programs/goto_program.cpp

```diff
@@ -516,6 +519,15 @@ std::ostream &goto_programt::output(
   return out;
 }
 
+/// Compute the target numbers for the targets in the goto program.
```

Comment:
How about: "Assign each control-flow join point in the goto program a unique index."